tests/integration/cli/restart_test.py

```diff
@@ -0,0 +1,71 @@
+import datetime
+import os
+import time
+
+import dateutil.parser
+from nose.tools import nottest
+
+from dusty.compiler.spec_assembler import get_all_repos
+from dusty.source import Repo
+from ...testcases import DustyIntegrationTestCase
+from ...fixtures import specs_fixture_with_depends
+
+class TestRestartCLI(DustyIntegrationTestCase):
+    def setUp(self):
+        super(TestRestartCLI, self).setUp()
+        specs_fixture_with_depends()
+        for repo in get_all_repos(include_specs_repo=False):
+            self._set_up_fake_local_repo(path=repo.remote_path)
+        self.run_command('bundles activate bundle-a bundle-b')
+        self.run_command('up')
+        time.sleep(1)
```

Comment:
It gives many milisecond digits, but on at least one occasion this method was unreliable without the sleep.  Perhaps `run_command('up')` can return before the start time per container is set?  I could look into that possibility